projects/packages/videopress/src/client/block-editor/blocks/video/components/player/types.native.ts

```diff
@@ -0,0 +1,5 @@
+export type NativePlayerProps = {
```

Comment:
Nitpick: Now that we have a file for native types, probably we no longer need to specify that the type is native by adding the `Native` word, WDYT?
```suggestion
export type PlayerProps = {
```